src/proto/FIO.proto

```diff
@@ -45,9 +45,27 @@ message Action {
         // Note: actor is not needed, computed from owner_fio_public_key
     }
 
+    // Action for transfering FIO coins; transfer_tokens_pub_key
+    message TransferTokens {
+        // FIO address of the payee. Ex.: "FIO6m1fMdTpRkRBnedvYshXCxLFiC5suRU8KDfx8xxtXp2hntxpnf"
+        string payee_public_key = 1;
```

Comment:
Can you transfer to a specific name and not public key?

Reply:
That's what FIO is all about

Reply:
With two operations you can (lookup, transfer).  This is FIO token transfer; the more frequent use case is transfer of other coin, and that is also 2 operations: look up name on FIO chain, perform transfer (on other chain).